docs/source/basics/latch_develop_example.md

```diff
@@ -0,0 +1,251 @@
+
+![Copy](../assets/latch-develop-example/copy.png)
+* To use a file on Latch as test data to the task, navigate to Latch Console, click on that file and copy the path using the "Copy" button on the sidebar. After copying the path, you can prefix it with `latch://` to specify that it is a file on Latch, and pass the whole string as a parameter to `LatchFile`. 
+
+## Calling the test script 
+* Inside `buggy-wf`, start a devleopment session with: 
+```
+$ latch develop .
+```
+Output: 
+```
+Copying your local changes...
+Could not find /Users/hannahle/Documents/GitHub/simple-variant-calling/buggy-wf/data - skipping
+Done.
+Successfully connected to remote instance.
+Pulling 6064_buggy-wf, this will only take a moment... 
+Image successfully pulled.
+>>>
+```
+
+* To run the test script, type: 
+```
+>>> run-script scripts/main.py
+```
+Output: 
+```
+Syncing your local changes... 
+Could not find /Users/hannahle/Documents/GitHub/simple-variant-calling/buggy-wf/data - skipping
+Finished syncing. Beginning execution and streaming logs:
+Finished downloading ecoli_rel606.fasta
+
+====================
+2022-11-17 00:09:13,104 flytekit ERROR Exception occured when executing task: [Errno 2] No such file or directory: 'bwa'
+====================
+```
+* The logs tell us that there is no file or directory called `bwa`. One potential reason why is we might not have installed the binary `bwa` correctly. 
+
+* To check this, let's open up a shell and check if the `bwa` command exists:
+```
+>>> shell
+
+account-4034-development@ip-10-0-11-243:~$ 
+```
+which opens up an interactive bash session. 
+```
+account-4034-development@ip-10-0-11-243:~$ bwa
+bash: bwa: command not found
+```
+* Indeed, our `bwa` binary was not installed properly! 
+* Checking the Dockerfile, we noticed that the installation instruction for `bwa` was commented out. Let's uncomment it.
+```Dockerfile
+...
+RUN apt install bwa
+...
+```
+
+* Because we made a modification to our Dockerfile, we have to rebuild the environment and enter a new development session to load in the newest changes. 
+* Exit your current development session: 
+```
+account-4034-development@ip-10-0-11-243:~$ exit
+>>> exit
+Exiting local development session
+```
+* Re-register your workflow with the new Docker image
+```
+$ latch register --remote .
+```
+* Once your workflow finishes registering, enter a new development session:
+```
+$ latch develop .
+```
+* Now, re-run the test script: 
+```
+>>> run-script scripts/main.py
+```
+Your script should now run successfully!
+
+**Where are my outputs?**
+* Let's inspect the return statement of the `_build_index` task inside `wf/__init__.py`:
+```python
+@small_task
+def _build_index(ref_genome: LatchFile = LatchFile("latch:///wgs/ref_genome/ecoli_rel606.fasta")) -> LatchDir: 
+    _bwa_cmd = [
+        "bwa", 
+        "index",
+        ref_genome.local_path
+    ]
+    subprocess.run(_bwa_cmd)
+    output = os.path.dirname(os.path.abspath(ref_genome.local_path))
```

Comment:
this is a python comment but why not use pathlib, its so much more readable

Reply:
```suggestion
    output = Path(ref_genome.local_path).resolve().parent
```

Reply:
thanks good point